src/cascadia/TerminalSettingsEditor/MainPage.xaml

```diff
@@ -88,6 +88,15 @@
 
             <muxc:NavigationViewItemHeader x:Uid="Nav_Profiles" />
 
+            <muxc:NavigationViewItem x:Name="BaseLayerMenuItem"
+                                     x:Uid="Nav_ProfileDefaults"
+                                     Tag="GlobalProfile_Nav"
+                                     Visibility="{x:Bind ShowBaseLayerMenuItem}">

```

Comment:
**IF** possible, bind `x:Load` instead.

Reply:
`x:Load` is not possible because of the dreaded "UnloadObject doesn't exist for some reason" error.